src/main/g8/src/main/java/com/lightbend/akka/http/sample/UserRoutes.java

```diff
@@ -70,8 +70,13 @@ private Route getUser(String name) {
           return rejectEmptyResponse(() ->
             onSuccess(
               () -> maybeUser,
-              performed ->
-                complete(StatusCodes.OK, (User) performed.get(), Jackson.<User>marshaller())
+              performed -> {
+                  if (performed.isPresent())
+                    return complete(StatusCodes.OK, performed.get(), Jackson.marshaller());
```

Comment:
Nice that this apparently doesn't need those explicit types after all :+1:

Reply:
Yes! :)